managed/devops/opscli/ybops/data/preflight_checks.sh

```diff
@@ -25,10 +25,13 @@ preflight_provision_check() {
 
   # Check for internet access.
   if [[ "$airgap" = false ]]; then
-    # Send 3 packets with 3 second timeout and return success if any succeed. Do not send
-    # multiple packets at once since ping will return an error if any packet fails.
+    # Attempt to run "/dev/tcp" 3 times with a 3 second timeout and return success if any succeed.
+    # --preserve-status flag will maintain the exit code of the "/dev/tcp" command.
+    HOST="yugabyte.com"
+    PORT=443
+
     for i in 1 2 3; do
-      ping -c 1 -W 3 www.yugabyte.com && break
+      timeout 3 bash -c "cat < /dev/null > /dev/tcp/$HOST/$PORT" --preserve-status && break
```

Comment:
This syntax would be better:
`/dev/tcp/${HOST}/${PORT}`

Reply:
Happy to help! I've applied the change above.